templates/default/capture-agent-manager-gunicorn-start.sh.erb

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+NAME="<%= @capture_agent_manager_name %>"
+USER="<%= @capture_agent_manager_usr_name %>"
+GROUP=${USER}
+FLASKDIR="/home/${USER}/sites/${NAME}"
+VENVDIR="${FLASKDIR}/venv"
+SOCKFILE="/home/${USER}/sock/${NAME}.sock"
+NUM_WORKERS=3
+
+echo "starting ${NAME}"
+
+# activate virtualenv
+source ${VENVDIR}/bin/activate
+
+export PYTHONPATH=${FLASKIDR}:${PYTHONPATH}
+
+# export all env vars for app
+source ${FLASKDIR}/${NAME}.env
```

Comment:
I'm not used to seeing `.env` files used in this way, but I think it's probably OK if makes things easier. 

Normally I would expect the `.env` to just list the `VARIABLE=foo` pairs (without the `export`), then your flask app would read/use the settings via something like https://github.com/grauwoelfchen/flask-dotenv